spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -170,6 +171,15 @@ def get_parser():
             Each label is a single voxel, which value corresponds to the spinal level (e.g.: 2 for spinal level 2). If you are using more than 2 labels, all spinal levels covering the region of interest should be provided (e.g., if you are interested in levels C2 to C7, then you should provide spinal level labels 2,3,4,5,6,7)."
         """)  # noqa: E501 (line too long)
     )
+    optional.add_argument(
+        '-lrootlet',
+        metavar=Metavar.file,
+        help=textwrap.dedent("""
+            Dorsal nerve rootlets segmentation from C2 to C8  Example: `anat_rootlets.nii.gz`
```

Comment:
I am a bit confused by the specification of "C2-C8". Is this because the PAM50 template only has those rootlets?

If so, maybe we should mention that in more detail. ("Only labels within the range C2-C8 are supported. If labels outside this range are provided, they will be ignored.") or something like that.

Reply:
Yes it is because the automatic rootlets segmentation is currently limited to C2-C8 (as the labels in the PAM50) (in sct_deepseg)

Reply:
added in : https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4647/commits/01721239c686904d3c99ded1d22759f9623077a1